.github/workflows/external.yml

```diff
@@ -0,0 +1,70 @@
+name: External-CI
+
+on: 
+  pull_request:
+
+jobs:
+  industrial_ci_external:
+    strategy:
+      matrix:
+        include:
+          - repo: 'ros-industrial/industrial_core'
+            ref: 'kinetic-devel'
+            env: {ROS_DISTRO: kinetic}
```

Comment:
I didn't include the repo destination and the ref directly inside the environment variable.